airflow/providers/amazon/aws/utils/utils.py

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Could this file like `helpers.py` or something like than.

Might be it is personal but `from airflow.providers.amazon.aws.utils.utils import <placeholder>` looks like that some stuff duplicated

Reply:
It is. Agree. I hate utils name, I usually use "common" and then each "comon" part should use a very specific package explaining what it is for. 

The "utils" and especially "utils.utils" says exactly NOTHING about what is going on there without actuallly looking what is in.